RevenueCatUI/Data/Constants.swift

```diff
@@ -12,10 +12,12 @@ enum Constants {
 
     static let defaultAnimation: Animation = .easeInOut(duration: 0.2)
     static let fastAnimation: Animation = .easeInOut(duration: 0.1)
-    static let displayAllPlansAnimation: Animation = .easeInOut(duration: 0.2)
+    static let toggleAllPlansAnimation: Animation = .spring(response: 0.35, dampingFraction: 0.7)
 
     static let defaultCornerRadius: CGFloat = 20
 
+    static let maxFooterHeight: CGFloat = 400
```

Comment:
Leaving unlimited height for footer felt bad so just putting this at 400 for now? But not sold on this at all 🙃